pkg/operator/ceph/cluster/cluster.go

```diff
@@ -595,6 +600,47 @@ func (c *cluster) reportTelemetry() {
 	telemetry.ReportKeyValue(c.context, c.ClusterInfo, telemetry.ExternalModeEnabledKey, strconv.FormatBool(c.Spec.External.Enable))
 }
 
+func (c *cluster) generateMonAssimilateConf(clientName string, settings map[string]string) error {
+	root := path.Join(c.context.ConfigDir, c.Namespace)
+	assimilateConfPath := path.Join(root, fmt.Sprintf("%s.txt", clientName))
+	if err := os.MkdirAll(root, 0744); err != nil {
+		return errors.Wrapf(err, "failed to create assimilateConfPath for %s.", clientName)
+	}
+
+	cfg := `
+[` + clientName + `]
```

Comment:
Do you need to write this `[global]` header to the file if you also have the `configFile.NewSection(clientName)` below? It seems like you would get two `[global]` headers in that case.

Reply:
it had only one section but anyway, makes sense to write only once.